src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -326,12 +326,12 @@ void forceReconnectImpl(ForceReconnectOptions options) throws InterruptedExcepti
 
             // stop i/o
             try {
-                this.reader.stop(false).get(10, TimeUnit.SECONDS);
+                this.reader.stop(false).get(100, TimeUnit.MILLISECONDS);
             } catch (Exception ex) {
                 processException(ex);
             }
             try {
-                this.writer.stop().get(10, TimeUnit.SECONDS);
+                this.writer.stop().get(100, TimeUnit.MILLISECONDS);

```

Comment:
On my end this is not an issue and the `Future` already clears very quickly.
So I don't think this contributes to the fix.

We could still lower the values of course, but then we'd probably need to make `closeSocketImpl` consistent with that as well?
Currently `closeSocketImpl` has:
```java
        try {
            this.reader.stop().get(10, TimeUnit.SECONDS);
        } catch (Exception ex) {
            processException(ex);
        }
        try {
            this.writer.stop().get(10, TimeUnit.SECONDS);
        } catch (Exception ex) {
            processException(ex);
        }
```